repository/filtering.go

```diff
@@ -0,0 +1,74 @@
+// Package repository provides filter model
+package repository
+
+import (
+	"database/sql"
+
+	"github.com/google/uuid"
+)
+
+type Postgres struct {
+	db *sql.DB
+}
+
+// NewPostgres returns Postgres with db
+func NewPostgres(db *sql.DB) *Postgres {
+	return &Postgres{db: db}
+}
+
+type FiltersGUD interface {
+	GetFilters() ([]Filter, error)
+	UpdateFilter() error
+	DeleteFilter() error
+}
+
+const (
+	addQuery    = "INSERT INTO FILTERS(name,type,configutation,reversed,user_id) VALUES ($1,$2,$3,$4,$5)"
+	getQuery    = "SELECT * FROM FILTERS WHERE user_id=$1"
+	updateQuery = "UPDATE FILTERS SET (type,configutation,reversed) = ($1,$2,$3) WHERE name = $4"
+	deleteQuery = "DELETE FROM FILTERS WHERE name=$1"

```

Comment:
So this query will delete all filters with given names for all users?

Reply:
Oh yeah, I should also point the id of user, thank you